index.js

```diff
@@ -8,7 +8,6 @@ const permalinks = require('metalsmith-permalinks');
 const assets = require('metalsmith-assets');
 const dataLoader = require('metalsmith-data-loader');
 const watch = require('metalsmith-watch');
-const branch = require('metalsmith-branch')
```

Comment:
this is usually used to have separate pipeline for stuff. in our case we would actually want something like:

```
const AssetBranch = branch();
const ContentBranch = branch();

AssetBranch.use(filter((file) => isScss(file) || isJs(file))) // <----
AssetBranch.use(assetProcessors)

ContentBranch.use(filter((file) => isMarkdown(file) || isPug(file))) // <----
AssetBranch.use(contentProcessors)

Metalsmith.use(AssetBranch)
Metalsmith.use(ContentBranch)
```

as we don't have those filtering statements, it seems safe to assume that the processors filter our the files they need to process themselves and that we can put everything into a single pipeline. given that the output looks identically (to me) when applying this PR, that seems to hold true.

That's what all the changes including `MS`, `AB`, and `CB` that follow are about. In short: we put the `assets`-processor into the main-pipeline and joined the `watch` expressions.